src/content/security/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Ethereum security and scams
+description: Staying safe on Ethereum
+lang: en
+sidebar: true
+---
+
+## Ethereum security and scams {#introduction}
+
+With interest in cryptocurrencies growing, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.
+
+<Divider />
+
+## Web security 101 {#web-security}
+
+### Use strong passwords {#use-strong-passwords}
+
+[Over 80% of account hacks are a result of weak or stolen passwords](https://cloudnine.com/ediscoverydaily/electronic-discovery/80-percent-hacking-related-breaches-related-password-issues-cybersecurity-trends/). A long combination of characters, numbers and symbols is best to keep your accounts secure.
+
+A common mistake individuals make is using a combination of two to three common, related dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](https://en.wikipedia.org/wiki/Dictionary_attack).
+
+```md
+Example of a weak password: CuteFluffyKittens!
+
+Example of a strong password: ymv\*azu.EAC8eyp8umf
+```
+
+Another common mistake is using passwords that can be easily guessed or found out through [social engineering](<https://en.wikipedia.org/wiki/Social_engineering_(security)>). Including your mother's maiden name, the name's of your children or pets, or dates of birth in your password is not secure and will increase the risk of your password getting hacked.
```

Comment:
I found a non-wiki [article ](https://www.webroot.com/us/en/resources/tips-articles/what-is-social-engineering) about social engineering. It does self advertise at the end but otherwise it looks good.